UML editor/src/main/java/codecain/UMLMethodInfo.java

```diff
@@ -0,0 +1,105 @@
+package codecain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * The UMLMethodInfo class represents a method in a UML class.
+ * It stores the method name and a list of parameters.
+ */
+public class UMLMethodInfo {
+
+    /**
+     * The name of the method.
+     */
+    private String methodName;
+
+    /**
+     * The list of parameters for the method.
+     */
+    private List<String> parameters;
+
+    /**
+     * Constructs a UMLMethodInfo object with the specified method name and parameters.
+     *
+     * @param methodName the name of the method
+     * @param parameters the list of parameters for the method (can be null)
+     */
+    public UMLMethodInfo(String methodName, List<String> parameters) {
+        this.methodName = methodName;
+        if (parameters != null) {
+            this.parameters = new ArrayList<>(parameters);
+        } else {
+            this.parameters = new ArrayList<>();
+        }
+    }
```

Comment:
The parameters will never be null because the command wont work if parameters are left empty.

Reply:
Thought process was I had two constructors initially that handled if we wanted to just to create the name, or name and parameters, but this simplifies it so we could use the same constructor but put null in the parameter for example if we wanted to just initialize a method, or if a method has no parameters we put null instead of inserting a string that is blank.